org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -115,16 +116,18 @@ class Translator {
      * and on the hub translator (this) for verification.
      */
     _subscribe(subscriptionInfo) {
-        var requestPath = '/subscription/' + subscriptionInfo.controlId;
-        return this._makeRequest(subscriptionInfo.endpointUri, requestPath, 'POST', '');
+        var requestPath = this._subcriptionPath + '?deviceId=' + subscriptionInfo.controlId 
+                                            + '&subscriptionURL=' + subscriptionInfo.callbackUrl;
+        return this._makeRequest(subscriptionInfo.endpointUri, requestPath, 'POST', "");
```

Comment:
I was told that whatever call the subcribe would include callbackUrl in the subscriptionInfo.

---
In reply to: [109980337](https://github.com/openT2T/translators/pull/225#discussion_r109980337) [](ancestors = 109980337)